src/main/java/games/strategy/engine/framework/headlessGameServer/AvailableGames.java

```diff
@@ -58,12 +57,19 @@ public GameData getGameData(final String gameName) {
     return getGameDataFromXML(m_availableGames.get(gameName));
   }
 
-  public URI getGameURI(final String gameName) {
```

Comment:
This method is not used, so I removed it.  It does not affect the functional changes encapsulated in this PR, but I figured due to its close proximity and related behavior to the changed code, it was a good candidate for removal.